1a.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector> 
+using namespace std;
+
+int main()
+{
+	vector<int> v1(5);
+	cout <<"Размер вектора: " <<v1.size()<<"\n";
+	cout <<"Коллекция чисел: \n";
+	int i;
+	for(i=4; i>=0; i--)
+	{
+		int c=0;
```

Comment:
What is the purpose of this variable? Is it a counter? And if so why is redefined every iteration of the cycle?